crates/turbopack/src/module_options/mod.rs

```diff
@@ -318,12 +321,30 @@ impl ModuleOptionsVc {
             ),
         ];
 
-        if enable_mdx {
+        if enable_mdx || enable_mdx_rs {
+            let (jsx_runtime, jsx_import_source) = if let Some(enable_jsx) = enable_jsx {
+                let jsx = enable_jsx.await?;
+                (jsx.runtime.clone(), jsx.import_source.clone())
+            } else {
+                (None, None)
+            };
+
+            let mdx_transform_options = MdxTransformOptionsVc::cell(MdxTransformOptions {
```

Comment:
Nit: you should be able to do `MdxTransformOptions {}.cell()` instead of `MdxTransformOptionsVc::cell(MdxTransformOptions {})`